clients/privacy-center/types/api/models/PrivacyExperienceResponse.ts

```diff
@@ -4,19 +4,50 @@
 
 import type { ComponentType } from "./ComponentType";
 import type { ExperienceConfigResponse } from "./ExperienceConfigResponse";
+import type { ExperienceMeta } from "./ExperienceMeta";
 import type { PrivacyNoticeRegion } from "./PrivacyNoticeRegion";
 import type { PrivacyNoticeResponseWithUserPreferences } from "./PrivacyNoticeResponseWithUserPreferences";
+import type { TCFFeatureRecord } from "./TCFFeatureRecord";
+import type { TCFPurposeConsentRecord } from "./TCFPurposeConsentRecord";
+import type { TCFPurposeLegitimateInterestsRecord } from "./TCFPurposeLegitimateInterestsRecord";
+import type { TCFSpecialFeatureRecord } from "./TCFSpecialFeatureRecord";
+import type { TCFSpecialPurposeRecord } from "./TCFSpecialPurposeRecord";
+import type { TCFVendorConsentRecord } from "./TCFVendorConsentRecord";
+import type { TCFVendorLegitimateInterestsRecord } from "./TCFVendorLegitimateInterestsRecord";
+import type { TCFVendorRelationships } from "./TCFVendorRelationships";
 
 /**
  * An API representation of a PrivacyExperience used for response payloads
  */
 export type PrivacyExperienceResponse = {
   region: PrivacyNoticeRegion;
   component?: ComponentType;
+  /**
+   * The Experience copy or language
+   */
   experience_config?: ExperienceConfigResponse;
   id: string;
+  tcf_purpose_consents?: Array<TCFPurposeConsentRecord>;
+  tcf_purpose_legitimate_interests?: Array<TCFPurposeLegitimateInterestsRecord>;
+  tcf_special_purposes?: Array<TCFSpecialPurposeRecord>;
+  tcf_features?: Array<TCFFeatureRecord>;
+  tcf_special_features?: Array<TCFSpecialFeatureRecord>;
+  tcf_vendor_consents?: Array<TCFVendorConsentRecord>;
+  tcf_vendor_legitimate_interests?: Array<TCFVendorLegitimateInterestsRecord>;
+  tcf_vendor_relationships?: Array<TCFVendorRelationships>;
+  tcf_system_consents?: Array<TCFVendorConsentRecord>;
+  tcf_system_legitimate_interests?: Array<TCFVendorLegitimateInterestsRecord>;
+  tcf_system_relationships?: Array<TCFVendorRelationships>;
```

Comment:
How do you generate these?

Reply:
I typically:
1. tag `fides`
2. Use that version in fidesplus
3. in admin-ui, run `npm run openapi:generate`, then `npm run format`. This will autogenerate all the TS types based off of the backend's `openapi.json` file. we run it off of fidesplus since fidesplus has a superset of fides types
4. that sets admin-ui up nicely for TS types (which I did in the branch this PR is based off of)
5. unfortunately for apps other than admin-ui, we manually copy and paste the relevant files over 🥲 there should be a way to share types one day, but that's been an [open issue](https://github.com/ethyca/fides/issues/3400) for a while now

Reply:
ahh thank you, I'll save this somewhere 👍